tasks/config/run.js

```diff
@@ -149,6 +149,7 @@ module.exports = function (grunt) {
         '--server.port=' + kibanaTestServerUrlParts.port,
         '--elasticsearch.url=' + esTestConfig.getUrl(),
         '--dev',
+        '--dev_mode.enabled=false',
```

Comment:
What's this for?

Reply:
I actually wonder if any of the devs ever uses that Debug spy panel :D If not we could also get rid of it completely.